packages/prettier-config/index.json

```diff
@@ -1,15 +1,5 @@
 {
   "$schema": "https://json.schemastore.org/prettierrc.json",
-  "arrowParens": "avoid",
-  "endOfLine": "auto",
-  "printWidth": 120,
-  "quoteProps": "preserve",
-  "tabWidth": 4,
-  "trailingComma": "none",
-  "overrides": [
-    {
-      "files": ["*.json", "*.jsonc", "*.json5", "*.yml"],
-      "options": { "tabWidth": 2 }
-    }
-  ]
+  "printWidth": 100,
```

Comment:
Out of curiosity, why 100 over 80/120. Happy medium? 👍🏼In any case, I like this because 120 sometimes doesn't fit when pinning a window to half of the screen (with a normal sized side-bar) at my current resolution.

Reply:
@turnertime 

When I was working on Mobile, 120 was too much, the lines got too long. At the same time when I used the prettier default 80 when submitting bug reports to Ionic, it felt too eager to wrap. So yes, 100 is a happy medium, plus I like that JohnG agrees that 80 is too short and is using 100 as well.

BTW I configured by VSCode to show two guide lines, one at 80 columns and another at 100 columns. I like to wrap my lines in the space between 80 and 100 even when writing code that is not explicitly wrapped by prettier

<img width="934" alt="image" src="https://github.com/pandell/frontend-config/assets/42297/7fe2f9fc-25da-4c29-93a0-b12301bbbf85">

Reply:
Thanks for the explanation. And good tip adding two guide lines. D'you do the same in Rider?

Reply:
@turnertime 

Yes sir:
<img width="752" alt="image" src="https://github.com/pandell/frontend-config/assets/42297/50911ce6-7af0-4c9f-bf72-0160ffe8c8be">